test/blocks/global-navigation/ch_de/breadcrumbs-from-file.test.html

```diff
@@ -6,20 +6,39 @@
     import { readFile } from '@web/test-runner-commands';
     import { expect } from '@esm-bundle/chai';
     import breadcrumbs from '../../../../libs/blocks/global-navigation/features/breadcrumbs/breadcrumbs.js';
-
+    import { setConfig } from '../../../../libs/utils/utils.js';
     runTests(() => {
       describe('breadcrumbs', () => {
         it('should create a breadcrumb from the url', async () => {
-          document.head.innerHTML = '<meta name="breadcrumbs-from-url" content="on">';
+          setConfig({
+            locales: {
+              '': { ietf: 'en-US', tk: 'hah7vzn.css' },
+              ch_de: {
+                ietf: 'de-CH',
+                tk: 'vin7zsi.css',
+                prefix: '/ch_de',
+                region: 'ch',
+              },
+            },
+          });
+          document.head.innerHTML =
+            '<meta name="breadcrumbs-from-url" content="on">';
           const breadcrumb = await breadcrumbs();
-          expect(breadcrumb.querySelectorAll('li').length).to.equal(5);
+          expect(breadcrumb.querySelectorAll('li').length).to.equal(4);
+
+          breadcrumb.querySelectorAll('li').forEach((li) => {
+            expect(li.textContent.trim()).to.not.equal('ch_de');
+          });
+
           // The 'global-navigation' parent folder has a hyphen, it should be removed
-          const gnavBreadcrumb = breadcrumb.querySelector('a[href $= "/global-navigation"]');
+          const gnavBreadcrumb = breadcrumb.querySelector(
+            'a[href $= "/global-navigation"]'
+          );
           expect(gnavBreadcrumb.textContent).to.equal('global navigation');
         });
 
-        it('should not create a breadcrumb from the url if the config isn\'t on', async () => {
-          document.head.innerHTML = ''
+        it("should not create a breadcrumb from the url if the config isn't on", async () => {
```

Comment:
I see the test on L12 uses single quotes, is there a reason for diverging from that?

Reply:
Prettier ran over it! I'll revert those changes